src/components/LiderbordCardComponent.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import Button from "@mui/material/Button";
+import Divider from "@mui/material/Divider";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Liderbord from "../model/liderbord";
+
+const bull = (
+  <Box
+    component="span"
+    sx={{ display: "inline-block", mx: "2px", transform: "scale(0.8)" }}
+  >
+    •
+  </Box>
+);
+
+function LiderbordCard({ liderbord }: { liderbord: Liderbord }) {
+  return (
+    <React.Fragment>
+      <Card
```

Comment:
Here I recommend you use the new `ContainerCard`from teh latest PR to remain consistant with styles on the figma and the overall app. It come with the appropriate padding shadow etc..